### ISSUE TITLE
why CalendarDay's toString() add 1 to month?

### ISSUE THREAD

**Issue by lusfold** [prolificinteractive/material-calendarview#142]
what this "this.month + 1" for?
```java
public String toString() {
        return String.format(Locale.US, "CalendarDay{%d-%d-%d}", new Object[]{Integer.valueOf(this.year), Integer.valueOf(this.month + 1), Integer.valueOf(this.day)});
    }
```

**Comment by dandc87** [comment]
I think this is a just a holdover from an old implementation that would return "user-friendly" strings. But now that it's "developer-friendly", I can see how this could be confusing while debugging. 

Also, I forget how varargs compile :mask: 

I'll fix these for the 1.0 release.

**Comment by lusfold** [comment]
alright,thx
